go/channelling/roomworker.go

```diff
@@ -290,6 +271,11 @@ func (r *roomWorker) Leave(sessionID string) {
 		r.mutex.Lock()
 		if _, ok := r.users[sessionID]; ok {
 			delete(r.users, sessionID)
+			if len(r.users) == 0 {
+				r.mutex.Unlock()
+				r.expired <- true
```

Comment:
I wonder whether we really need the changes to this file now that we store a room's PIN in localStorage.